libraries/oneview_helper.rb

```diff
@@ -18,8 +18,8 @@ module Helper
     # @param context Context from the resource action block (self)
     # @param [String, Symbol] type Name of the resource type. e.g., :EthernetNetwork
     # @param [String, Symbol] action Action method to call on the resource. e.g., :create_or_update
-    def self.do_resource_action(context, type, action)
-      klass = OneviewCookbook::Helper.get_resource_class(context, type)
+    def self.do_resource_action(context, type, action, secondary_api = nil)
```

Comment:
All these references to 'secondary_api' are pretty confusing to me, partly because when it's set, it's actually the only api module, not a secondary one. How about we name it something 'base_module'? And I think it makes the most sense to set the default to `OneviewCookbook` instead of nil. That way it's clear what the default is. Also, add the `@param` docs describing it and accepted value(s).